### ISSUE TITLE
Accessibility: Indicators for decorative images

### ISSUE THREAD

**Issue by michaelvcolianna** [bluesky-social/social-app#6845]
### Describe the Feature

Similar to #645 (particularly, the reply from cygaj), it would be nice to have the option of marking an image as, "decorative." Some possible use cases:

- An image is described in the text of the post
- An image accompanies a post but is not a visual reference required to understand the post

If marked, "decorative," the image's ALT text could either just read, "decorative," or be left empty. And instead of the ALT badge (or lack thereof) the image could have a new one stating, "decorative," or similar.

This would be an important addition because it shows that the user thought about accessibility, whereas it may not be obvious when an image is lacking ALT text.

### Attachments

![A post on the SFFChat hashtag asking users to tell about themselves and their work, with an image re-stating that.](https://github.com/user-attachments/assets/f1a53b40-e675-4d0b-9219-95eb07581b69)
The primary import of the image in this post is restating the post. Its ALT text is "Illustration of a woodland scene at night. An astronaut and a wizard stand looking at each other. Purple sparkly dust swirls up from the text." This ALT text isn't bad but it also isn't really necessary.

![A post from a person who is playing through Final Fantasy 6 and found a guide book helpful. They included a picture of the book and a Game Boy style emulation device playing the game.](https://github.com/user-attachments/assets/5dde3860-0449-43e8-8ba7-68eddc650295)
The image in this post doesn't have ALT text and doesn't need it since the absence of the image wouldn't hinder the reader's understanding. The user's intent is difficult to discern, however - is it a feature they aren't aware of or did they realize the image doesn't need ALT text?
